pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -140,7 +141,12 @@ func (pl *VolumeBinding) podHasPVCs(pod *v1.Pod) (bool, error) {
 		hasPVC = true
 		pvc, err := pl.PVCLister.PersistentVolumeClaims(pod.Namespace).Get(pvcName)
 		if err != nil {
-			// The error has already enough context ("persistentvolumeclaim "myclaim" not found")
+			// The error usually has already enough context ("persistentvolumeclaim "myclaim" not found"),
+			// but we can do better for generic ephemeral inline volumes where that situation
+			// is normal directly after creating a pod.
+			if ephemeral && apierrors.IsNotFound(err) {
+				err = fmt.Errorf("waiting for ephemeral volume controller to create the persistentvolumeclaim %q", pvcName)
```

Comment:
is it always the case that this type of PVC is created by the ephemeral volume controller?

Reply:
In practice, yes. It would be possible to set up a cluster where that controller is disabled and some other entity creates the PVCs, but that seems very unlikely.